jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -127,6 +128,11 @@ public void buildDocker()
               + ex.getInvalidPathValue(),
           ex);
 
+    } catch (InvalidPlatformConfigurationException ex) {
+      throw new GradleException(
+          "container.platforms contains a null architecture or os name: " + ex.getInvalidPlatform(),
```

Comment:
How about `from.platform misses a required field or has an invalid value: `?

Note it's `from.platform`.

Reply:
Gotcha